src/main/java/org/jenkinsci/test/acceptance/junit/AbstractClassBasedJUnitTest.java

```diff
@@ -0,0 +1,37 @@
+package org.jenkinsci.test.acceptance.junit;
+
+import org.jenkinsci.test.acceptance.po.CapybaraPortingLayerImpl;
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.junit.ClassRule;
+import org.openqa.selenium.WebDriver;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * Convenience base class to derive your plain-old JUnit tests from when using {@link JenkinsAcceptanceTestRule} as a {@link ClassRule}.
+ *
+ * @author Andres Rodriguez
+ */
+public class AbstractClassBasedJUnitTest extends CapybaraPortingLayerImpl {
+
+    @ClassRule
+    public static JenkinsAcceptanceTestRule rules = JenkinsAcceptanceTestRule.classRule();
+
+    /**
+     * Jenkins under test.
+     */
+    @Inject
+    public static Jenkins jenkins;
```

Comment:
It is assigned per-class for all the tests in the class, that being the idea of a `ClassRule`.

Please take into account that the mechanism provided in this PR does not substitute the existing `AbstractJUnitTest`-based tests but complement the approach for those cases *specifically designed* to work at class level. The use cases for it are:
* The main one when using the `existing` driver (tests designed specifically for the case in which the JUT is an external one, so they are staeful).
* Tests classes designed so that the different methods don't interfere with each other, as is the case with the modified `FolderPluginTest`.

In both cases they get to share initialization code.